server/controllers/statistics.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response, NextFunction } from 'express';
+import Family from '../database/models/family';
+import Doners from '../database/models/donor';
+import Donations from '../database/models/donation';
+import { sequelize } from '../database/models';
+
+const statistics = async (req: Request, res: Response, next:NextFunction) => {
+  try {
+    const [families, doners, donations] = await Promise.all([
+      Family.count(), Doners.count(),
+      Donations.findAll({
+        attributes: [[sequelize.fn('SUM', sequelize.col('money')), 'money'],
+          [sequelize.fn('SUM', sequelize.col('food')), 'food'],
+          [sequelize.fn('SUM', sequelize.col('clothes')), 'clothes']],
```

Comment:
Dont think these lines will cause   a problem when the value of one of them is null? as it could be null.
Maybe we can edit the madels.

Reply:
I can set the default value in dontaions table for (food,clothes,money) to zero ?